spec/classes/nginx_spec.rb

```diff
@@ -337,6 +337,12 @@
 
           describe 'nginx.conf template content' do
             [
+              {
+                title: 'should not set load_module',
```

Comment:
can you add another test that sets a module and checks if it is present in the config file?

Reply:
This is done, check diff below.

Reply:
This is done, check lines 891-910